sdk/identity/azure-identity/tests/test_environment_credential.py

```diff
@@ -34,19 +34,19 @@ def test_incomplete_configuration():
 
 
 @pytest.mark.parametrize(
-    "credential_name,environment_variables",
+    "credential_name,envvars",
```

Comment:
Why we want to rename it?

Reply:
`environment_variables` is also a [fixture](https://github.com/Azure/azure-sdk-for-python/blob/e0fc7d1a5848528a2a2aff72820bb9290f03a31f/tools/azure-sdk-tools/devtools_testutils/proxy_fixtures.py#L108) that I use, and when we try to use the same name as a parameter, pytest gets confused and gives me errors.